bazel/scripts/run_sudo_tests.sh

```diff
@@ -150,6 +150,16 @@ fi
 
 cd "${MAGMA_ROOT}"
 
+# Build the dhcp_helper_cli script and create a symlink for its intended usage.
+# This is needed for the tests
```

Comment:
imho the info for what tests this is needed is prone to be outdated - I would remove it

Reply:
Right, I removed the info about the tests and left it generic.